src/components/shared/DownloadButton.tsx

```diff
@@ -42,22 +43,40 @@ export default function DownloadButton(props: DownloadButtonProps) {
     }
     
     return (
-        <Button key={buttonLabelKey} size={buttonSize} className="download-data-btn">
+        iconName === "" ?       
+        <Popup content={Translate(popupText)} trigger={
+            <Button key={buttonLabelKey} size="large" iconName={iconName}className="download-data-btn mb-2 mr-2">
```

Comment:
Again not a big deal for now but we should be using the buttonSize prop here.

Also I see that we've defined some hardcoded width and heights, ideally we should have some mapping from iconSize to height/width and allow users to specify iconSize. That's more of a nitpick tho